lsp/src/htmx/attributes/hx-ext.md

```diff
@@ -0,0 +1,8 @@
+reference an extension
```

Comment:
I noticed https://github.com/microsoft/vscode-html-languageservice links the source of each completion with the source material like:
![Screenshot 2023-08-28 at 09 33 12](https://github.com/ThePrimeagen/htmx-lsp/assets/3959744/39cf6321-d01c-4067-85bc-1bd1f0d84b00)

Shall we do the same? WDYT?

Reply:
I think this might be a good idea, will add those in a bit.